src/stylesheets/App.css

```diff
@@ -28,3 +28,14 @@
 .nav-link-bold {
   font-weight: bold;
 }
+
+.checkbox-wrapper {
+  display: flex;
+  justify-content: center;
+}
+
+li {
+  list-style-type: none;
+  margin-left: 0.5rem;
+  margin-bottom: 0.5rem;
+}
```

Comment:
Entering CSS world! And yay for relative units vs px. 

Suggestion (no problem if you aren't feeling it): include a border and a light background-color for the `li`, something simple like
  `border-style: solid;`
  `border-color: fuchsia;`
  `background-color: lightgray;`

It is a quick default I've just started including to help my eyes see how the sections laid out, even if it looks hideous in the production stage :)

Reply:
Honestly, I can't wait to style  the heck out of this app! But I've only added any CSS that helps improve the functionality/understanding the new features for now. (Like removing the bullet points from the `<li>` and placing the checkboxes next to the list item and not above/below them).

That said, I'll definitely remember your layout tip!!